include/fmt/format.h

```diff
@@ -1276,8 +1276,6 @@ template <> struct float_info<float> {
   static const int small_divisor = 10;
   static const int min_k = -31;
   static const int max_k = 46;
-  static const int divisibility_check_by_5_threshold = 39;
-  static const int case_fc_pm_half_lower_threshold = -1;
```

Comment:
Should these be removed from `float_info<double>` as well?

Reply:
Ha. Another careless mistake. Sorry about that, it's fixed now.